operator/pkg/helm/helm.go

```diff
@@ -42,7 +42,8 @@ import (
 // Render produces a set of fully rendered manifests from Helm.
 // Any warnings are also propagated up.
 // Note: this is the direct result of the Helm call. Postprocessing steps are done later.
-func Render(namespace string, directory string, iop values.Map, kubernetesVersion *version.Info) ([]manifest.Manifest, util.Errors, error) {
+// TODO: ReleaseName hardcoded to istio, this should probably have been the component name. However, its a bit late to change it.
```

Comment:
nit: maybe move the TODO to where we actually hard-code the value? (ie istioctl/pkg/tag/generate.go:212)

Reply:
There is more than one place that uses hard code, which is why I put it here.

Reply:
yeah, that will make the helm render result always name as `istio` with a gateway chart.

some context for this:
- https://github.com/istio/istio/pull/55323
- https://github.com/istio/istio/pull/55133

Reply:
I see, thank you. Works for me